qdrant-landing/content/articles/discovery-search.md

```diff
@@ -0,0 +1,95 @@
+small_preview_image: /articles_data/discovery-search/icon.svg # This image will be used in the list of articles at the footer, should be 40x40px
+preview_dir: /articles_data/discovery-search/preview # This directory contains images that will be used in the article preview. They can be generated from one image. Read more below. Required.
+weight: -100 # This is the order of the article in the list of articles at the footer. The lower the number, the higher the article will be in the list.
+author: Luis Cossío # Author of the article. Required.
+author_link: https://coszio.github.io # Link to the author's page. Required.
+date: 2023-12-11T13:00:00+03:00 # Date of the article. Required.
+draft: false # If true, the article will not be published
+keywords: # Keywords for SEO
+  - why use a vector database
+  - specialty
+  - search
+  - discovery
+  - state of the art
+  - vector search
+---
+
+When Christopher Columbus and his crew sailed to cross the Atlantic Ocean, they were not looking for America. They were looking for a new route to India, and they were convinced that the Earth was round. They didn't know anything about America, but since they were going west, they stumbled upon it.
+
+They couldn't reach their target, because the context of the geography didn't let them, but once they realized it wasn't India, they claimed it a new "discovery" for their crown. Let's keep this concepts of target and context in mind, as we explore the new functionality of Qdrant: __Discovery search__.
+
+In version 1.7, Qdrant [released](/articles/qdrant-1.7.x/) this novel API that lets you constrain the space in which a search is performed, relying only on pure vectors. This is a powerful tool that lets you explore the vector space in a more controlled way. It can be used to find points that are not necessarily closest to the target, but are still relevant to the search.
+
+One previous way to restrict the points that are available to the search is by filtering out the points we know for a fact are not relevant. This is really powerful because it allows us to craft complex filters that show only the points that satisfy their criteria deterministically. However, the data –or payload– associated to each point is arbitrary and does not tell us anything about their position in the vector space. In other words, it would create a _mask_ rather than a _boundary_ in the space.
+
+This is where the concept of __vector _context___ comes in. We define _context_ as a list of pairs, which in turn are made up of a positive and a negative vector. By adding a context, we can define boundaries within the vector space, to which the positive side will always be preferred over the negative side, effectively partitioning the space in which the search is performed. After the space is partitioned, we then need a _target_ to select the points that are closest to it.
```

Comment:
The word `boundary` is a bit fuzzy for higher dimension space.
Is it like a hyperplane cutting orthogonal between the positive and negative vector?

Reply:
Exactly! I like the way you are saying it